src/main/java/racingcar/utils/InputUtils.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.utils;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import racingcar.validator.InputValidator;
+
+public class InputUtils {
+
+    private static final String nameSeparator = ",";
```

Comment:
```suggestion
    private static final String NAME_SEPARATOR = ",";
```
일반적으로 상수는 대문자로 작성하며, 복합어는 언더스코어(`_)를 사용하여 단어를 구분합니다